test/functional/conftest.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
@jesterhazy, do you care about this being under functional or integration/sagemaker?

I think most of our framework containers are under integration/sagemaker, except for the sagemaker-tensorflow-container repository.

Reply:
yes, it seems a bit funky to have both. the stuff in test/integration are local docker container tests, so renaming test/functional to test/sagemaker or renaming everything to test/integration/{local, sagemaker} would fit the convention better

Reply:
I'll rename it to test/sagemaker